media/util.ts

```diff
@@ -2,29 +2,32 @@
 // Licensed under the MIT license.
 
 import { ByteData } from "./byteData";
-import { SelectHandler } from "./selectHandler";
 
 // Assorted helper functions
 
-
 /**
  * @description Class which represents a range of numbers
  */
 export class Range {
-    private start: number;
-    private end?: number;
+    public readonly start: number;
+    public readonly end: number;
     // Construct a range object representing [start, end] inclusive of both
     /**
      * @description Constructs a range object represneting [start, end] inclusive of both
```

Comment:
This wasn't you, but can you fix my spelling error of representing and remove the comment above that's an exact duplicate of the js doc comment.